packages/react-theme-sass/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@fluentui/react-theme-sass",
+  "version": "9.0.0-beta.3",
+  "description": "SASS variables referencing react-theme design tokens injected to DOM by react-provider.",
+  "private": true,
+  "main": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "style": "sass/tokens.scss",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build --commonjs",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "test": "jest",
+    "docs": "api-extractor run --config=config/api-extractor.local.json --local",
+    "build:local": "tsc -p ./tsconfig.lib.json --module esnext --emitDeclarationOnly && node ../../scripts/typescript/normalize-import --output ./dist/packages/react-theme-sass/src && yarn docs",
+    "storybook": "start-storybook",
+    "start": "yarn storybook",
+    "type-check": "tsc -b tsconfig.json"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "*",
+    "@fluentui/scripts": "^1.0.0"
+  },
+  "dependencies": {
+    "tslib": "^2.1.0"
+  },
```

Comment:
Since this package doesn't ship JS (if I'm understanding right) I don't think it should depend on tslib
```suggestion
```